setup.py

```diff
@@ -25,85 +25,116 @@
 import six
 
 if six.PY3:
-    exec(open('ipopt/version.py', encoding="utf-8").read())
+    exec(open("cyipopt/version.py", encoding="utf-8").read())
 else:
-    exec(open('ipopt/version.py').read())
+    exec(open("cyipopt/version.py").read())
 
-PACKAGE_NAME = 'ipopt'
+PACKAGE_NAME = "cyipopt"
+DEPRICATED_PACKAGE_NAME = "ipopt"
 VERSION = __version__
-DESCRIPTION = 'A Cython wrapper to the IPOPT optimization package'
+DESCRIPTION = "A Cython wrapper to the IPOPT optimization package"
 if six.PY3:
-    with open('README.rst', encoding="utf-8") as f:
+    with open("README.rst", encoding="utf-8") as f:
         LONG_DESCRIPTION = f.read()
 else:
-    with open('README.rst') as f:
+    with open("README.rst") as f:
         LONG_DESCRIPTION = f.read()
-AUTHOR = 'Matthias Kümmerer'
-EMAIL = 'matthias.kuemmerer@bethgelab.org'
+KEYWORDS = [
+    "optimization",
+    "nonlinear programming",
+    "ipopt",
+    "interior-point",
+    "nlp",
+    "coin-or",
+]
+AUTHOR = "Matthias Kümmerer"
+EMAIL = "matthias.kuemmerer@bethgelab.org"
 URL = "https://github.com/matthias-k/cyipopt"
-DEPENDENCIES = ['numpy', 'cython', 'six', 'future', 'setuptools']
+DEPENDENCIES = [
+    "numpy",
+    "cython",
+    "six",
+    "future",
+    "setuptools",
+]
+LICENSE = "EPL-1.0"
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: Eclipse Public License 1.0 (EPL-1.0)",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 2.7",
```

Comment:
Remove.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/74228be98587b24d86b6c4246018ff4a8f4206b1.